packages/plugin-rok/CHANGELOG.md

```diff
@@ -0,0 +1,7 @@
+# @jspsych-contrib/plugin-rok
```

Comment:
This file should be deleted because it will be automatically generated by the release process.